javascriptv3/example_code/libs/scenario/scenario-example.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Scenario,
+  ScenarioAction,
+  ScenarioInput,
+  ScenarioOutput,
+} from "./index.js";
+
+const greet = new ScenarioOutput(
+  "greet",
+  "Hi! This is a scenario. It can handle your " + "inputs and outputs.",
+);
+
+const describeInput = new ScenarioOutput(
+  "describe input",
+  "ScenarioInput collects three different types of input from the user: " +
```

Comment:
Why the multiline string with concatenation?

Reply:
Why at all? Readability.
Why instead of template literal? So the text can have proper indentation without affecting the string.